src/client.ts

```diff
@@ -13,6 +13,21 @@ import type { HttpClientConfig } from './http';
 
 const debug = createDebug('strapi:core');
 
+/**
+ * Mapping of well-known Strapi resource names to their plugin configurations.
+ * This enables automatic handling of special Strapi content-types that have
+ * different API contracts than regular content-types.
+ */
+const WELL_KNOWN_STRAPI_RESOURCES: Record<string, { plugin: { name: string; prefix: string } }> = {
+  // Users from users-permissions plugin don't wrap data and have no route prefix
+  users: {
+    plugin: {
+      name: 'users-permissions',
+      prefix: '',
+    },
+  },
+};
+
```

Comment:
Where is the best place for this? Opinion @Convly?

Reply:
We could also move there [this one](https://github.com/strapi/client/blob/main/src/content-types/collection/manager.ts#L12)